automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/Modals.scala

```diff
@@ -29,10 +50,12 @@ case class ErrorModal(implicit webDriver: WebDriver) extends OKCancelModal {
     testId("error-modal").element != null
   }
 
-  def getErrorText(): String = {
+  def getErrorText: String = {
     readText(testId("message-modal-content"))
```

Comment:
probably also a question for brian... do we want to remove the () when we have an empty parameter list? it seems we have left them in for most of the calls. but maybe it's more scala-y to remove them?

Reply:
Having the empty parens on a function that begins with "get" causes this warning in IntelliJ:

> Method with accessor-like name is empty-paren less... (⌘F1) 
>
> Methods that follow JavaBean naming contract for accessors are expected to have no side effects. The recommended convention is to use a parameterless method whenever there are no parameters and the method have no side effect. This convention supports the uniform access principle, which says that client code should not be affected by a decision to implement an attribute as a field or method. 
>
> * Refer to Programming in Scala, 10.3 Defining parameterless methods

I have left the empty parens for functions that wait for some state to change. While their only side effect is the passage of time, they observe a side effect (some UI state change). I could be persuaded otherwise but that's the convention I've been trying to follow.

Reply:
I like having empty parens for functions with side effects, and I would suggest naming them in such a way as to avoid that warning. "Get" really does imply a lack of side effects.